### ISSUE TITLE
SearchField: The x button should get a focus stop

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#4815]
**Describe the bug**
Add a tabbable setting so that the X button can get a tab when you have focus in the field. This should be true by default but the setting can make it un untabble again

**To Reproduce**
1. Go to https://master-enterprise.demo.design.infor.com/components/searchfield/example-index.html
2. Tab in and type something until the x appears
3. Tab -> this should hit the x

**Expected behavior**
X is tabbable for WCAG  / 508

**Version**
- ids-enterprise: [e.g. v4.37.0]

**Additional context**
See also https://github.com/infor-design/enterprise/issues/4127 for more fields...

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
